### ISSUE TITLE
[Question]: Is there a build config change needed to upgrade this lib from 13 to 14 without upgrading Angular to 14?

### ISSUE THREAD

**Issue by robertfwest** [damienbod/angular-auth-oidc-client#1537]
**What Version of the library are you using?**
14.1.4 (no problem with 13.1.0 but want the new feature of overriding redirectUrl)

**Question**
How can I get my app to compile without upgrading to Angular 14? (I'm using 13.3 and have several reasons to not upgrade yet.)

This is the error I'm getting (with some details X'd out):
```
./node_modules/angular-auth-oidc-client/fesm2020/angular-auth-oidc-client.mjs - Error: Module build failed (from ./node_modules/@angular-devkit/build-angular/src/babel/webpack-loader.js):
TypeError: Cannot create property 'message' on string '/home/X/projects/XX/XXX/node_modules/angular-auth-oidc-client/fesm2020/angular-auth-oidc-client.mjs: This application depends upon a library published using Angular version 14.1.0, which requires Angular version 14.0.0 or newer to work correctly.
Consider upgrading your application to use a more recent version of Angular.
  5045 | }
  5046 | AuthModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.1.0", ngImport: i0, type: AuthModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
> 5047 | AuthModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "14.1.0", ngImport: i0, type: AuthModule, imports: [CommonModule, HttpClientModule] });
       |                   ^^^^^^^^^^^^^^^^^^^^^^^^39m^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^39m^^^^^^^^^^^^^^^^^^
  5048 | AuthModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "14.1.0", ngImport: i0, type: AuthModule, imports: [CommonModule, HttpClientModule] });
  5049 | i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.1.0", ngImport: i0, type: AuthModule, decorators: [{
  5050 |             type: NgModule,'
    at run (/home/X/projects/XX/XXX/node_modules/@babel/core/lib/transformation/index.js:37:15)
    at run.next (<anonymous>)
    at Function.transform (/home/X/projects/XX/XXX/node_modules/@babel/core/lib/transform.js:25:41)
    at transform.next (<anonymous>)
    at step (/home/X/projects/XX/XXX/node_modules/gensync/index.js:261:32)
    at /home/X/projects/XX/XXX/node_modules/gensync/index.js:273:13
    at async.call.result.err.err (/home/X/projects/XX/XXX/node_modules/gensync/index.js:223:11)
    at /home/X/projects/XX/XXXi/node_modules/gensync/index.js:37:40

```

**Comment by lorenzodallavecchia** [comment]
@robertfwest try using version 14.0.2. It contains the new features of 14.0.0 but still works with Angular 13.

I think this is a subtle bug, not just a question. Commit 5ec347912d50afd970ee045c7c98979a68559666 migrated the sources to Angular 14, but by doing this the AOT metadata embedded in the compiled files started requiring Angular 14. However, the `package.json` published to npm declares support for Angular `>=13.0.0` which is not the case when using AOT.

Basically, version 14.1.0 of angular-auth-oidc-client introduced a breaking change despite it being a minor release.
